src/app.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import TodoListMain from './todoListMain';
+import ShowTodo from './showTodo';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <TodoListMain />
+        {/* <ShowTodo /> */}
```

Comment:
I know  a lot of sticklers that would give you a hard time for leaving in commented out code :). Especially cause you're still importing this component and not using it.